### ISSUE TITLE
Notify clients to change their API keys to new format - Email Ticket

### ISSUE THREAD

**Issue by yaelberger-commits** [cds-snc/notification-planning#784]
_Notify clients to change their API keys to new format_

## Description

As a current Notify user, I need to be aware that Notify has changed the format for API keys to include a prefix for security scanning so that I can change my API key to the new format before the grace period ends and my application breaks. 

WHY are we building?
We are adding a prefix to our API keys so we can scan for them in public repos for better security
We need to let current users know there will be a change and they will have to rotate their old API keys for new ones with the prefix before the grace period ends.

WHAT are we building?
An email notification to current clients

VALUE created by our solution
Enhanced security and scanning abilities to detect realtime system misuse and API keys that may be stored publicly 

## Acceptance Criteria** (Definition of done) 

Given a current Notify user has an API key, when current users with old API keys are made aware of the change then they rotate their API keys before the grace period ends for a seamless transition to the new format. 

- [x] Email EN/FR to all clients explaining the change, and steps they need to take
- [x] Email includes a path to contact us for help

* Security controls in place

## QA Steps

- [x] Send test emails to ourselves
- [x] Test in production rotating our own API keys to the new format 
- [x] QA the email to current users in both EN and FR for typos, translation and any broken links

**Comment by andrewleith** [comment]
TL;DR when we communicate with clients we should be explicit that they provide the entire string that is created when they generate their new API key, and not just the last 36 characters.  Our documentation will need to be updated as well.

--- 
There is a discrepancy between our guidance in the documentation oh how to call the API and the actual implementation in the client libraries. [Our documentation](https://documentation.notification.canada.ca/en/start.html#headers) indicates a user only needs to provide the secret ID in the header when calling the API.  While this is true, if a user instead uses any of the client libraries, the rule is different, and the entire string that is created when an API key is generated MUST be used (i.e. `keyname-serviceid-keyid`).

**Comment by YedidaZalik** [comment]
@yaelberger-commits is the card intended for the dev audience only?  Or are there non-devs who might also need this information?

**Comment by yaelberger-commits** [comment]
@YedidaZalik I don't think we know for sure. Service owners, security assessors, and other decision makers may also be audience for this info

**Comment by jzbahrai** [comment]
The email is ready for review from content.

**Comment by YedidaZalik** [comment]
Using the suggest function, I put some questions/changes in the document on Thursday, which Jumana responded to. I added some more changes and questions this morning.

**Comment by YedidaZalik** [comment]
Will connect with Jumana, either through comments function in the doc or a sync.

**Comment by YedidaZalik** [comment]
Worked on email in tandem with Jumana. Latest draft on second page: https://docs.google.com/document/d/19uD80moowqMVUPoHLh-5_lEy1ApleMF77L-iXPAGNaI/edit#heading=h.wh5cl5jetxnv

**Comment by YedidaZalik** [comment]
Sent to translation today in a clean copy: https://docs.google.com/document/d/19-uhCNzWs6uwSagHF6sQe6kKOSw6SJuq0tOttgWCdlQ/edit#heading=h.1933ypt3f3r8

**Comment by YedidaZalik** [comment]
Marie-Sophie will translate and Nick will proofread.

**Comment by YedidaZalik** [comment]
Document back from translation https://docs.google.com/document/d/1O7MWSIYBPbTFklt-x2qb7np65-aoNU-MDf2POClUp9U/edit
Incorporated into English email https://docs.google.com/document/d/10favFYKEc_hsP1pmdIu-A-todnPL8TsUJXI3SPYt5Hk/edit
Will ask Anik to review this mass communication

**Comment by anikbrazeau** [comment]
✅ Email reviewed. (cc @yaelberger-commits) 
🧪 Should be ready to test! (cc @jzbahrai) 

https://docs.google.com/document/d/10favFYKEc_hsP1pmdIu-A-todnPL8TsUJXI3SPYt5Hk/edit

**Comment by yaelberger-commits** [comment]
Emails were sent in Staging, typos were fixed, then tests were sent in Prod to a few folks on the team. Finally all first round of emails were sent yesterday in Prod and some canned responses were created for support.
